lang/uk/lang.php

```diff
@@ -0,0 +1,118 @@
+        'new' => 'Нова сторінка',
+        'add_subpage' => 'Додати підсторінку',
+        'invalid_url' => 'Некоректний формат URL. URL повинен починатися з прямого слеша і може містити цифри, латинські літери і такі символи: _-/.',
+        'url_not_unique' => 'Цей URL вже використовується іншою сторінкою.',
+        'layout' => 'Шаблон',
+        'layouts_not_found' => 'Шаблони не знайдені',
+        'saved' => 'Сторінка була успішно збережена.',
+        'tab' => 'Сторінки',
+        'manage_pages' => 'Управління сторінками',
+        'manage_menus' => 'Управління меню',
+        'access_snippets' => 'Доступ до сніппетів',
+        'manage_content' => 'Управління змістом',
+    ],
+    'menu' => [
+        'menu_label' => 'Меню',
+        'delete_confirmation' => 'Ви дійсно хочете видалити вибрані пункти меню?',
+        'no_records' => 'Меню не знайдені',
+        'new' => 'Нове меню',
+        'new_name' => 'Новое меню',
+        'new_code' => 'nove-menu',
+        'delete_confirm_single' => 'Ви дійсно хочете видалити це меню?',
+        'saved' => 'Меню було успішно збережено.',
+        'name' => "Ім`я",
+        'code' => 'Код',
+        'items' => 'Пункти меню',
+        'add_subitem' => 'Додати підменю',
+        'code_required' => "Поле Код обов'язкове",
+        'invalid_code' => 'Некоректний формат Коду. Код може містити цифри, латинські літери і такі символи: _-/'
+    ],
+    'menuitem' => [
+        'title' => 'Назва',
+        'editor_title' => 'Редагувати пункт меню',
+        'type' => 'Тип',
+        'allow_nested_items' => 'Дозволити вкладені',
+        'allow_nested_items_comment' => 'Вкладені пункти можуть бути динамічно згенеровані статичною сторінкою або іншими типами елементів',
+        'url' => 'URL',
+        'reference' => 'Посилання',
+        'title_required' => "Назва обов'язково",
+        'unknown_type' => 'Невідомий тип меню',
+        'unnamed' => 'Безіменний пункт',
+        'add_item' => 'Додати пункт (<u> i </u>)',
+        'new_item' => 'Новий пункт',
+        'replace' => 'Замінювати цей пункт його згенеруванними нащадками',
+        'replace_comment' => 'Відмітьте для перенесення згенерованних пунктів меню на один рівень з цим пунктом. Сам цей пункт буде приховано.',
+        'cms_page' => 'Сторінки CMS',
+        'cms_page_comment' => 'Оберіть відкриваючу при натисканні сторінку.',
+        'reference_required' => 'Необхідне посилання для пункту меню.',
+        'url_required' => 'Необхідний URL',
+        'cms_page_required' => 'Будь ласка, виберіть сторінку CMS',
+        'code' => 'Код',
+        'code_comment' => 'Введіть код пункту меню, якщо хочете мати до нього доступ через API.'
+    ],
+    'content' => [
+        'menu_label' => 'Зміст',
+        'cant_save_to_dir' => 'Збереження файлів змісту в директорію static-pages заборонено.'
+    ],
+    'sidebar' => [
+        'Add' => 'Додати',
+        'search' => 'Пошук...'
```

Comment:
Please make sure all elements have trailing commas

Reply:
I copied the original ru/lang file and they weren't there. But, ok. I corrected both